aptos-move/aptos-e2e-comparison-testing/src/main.rs

```diff
@@ -35,6 +36,29 @@ pub enum Cmd {
         /// Dump the write set of txns
         #[clap(long, default_value_t = false)]
         dump_write_set: bool,
+        /// Target account
+        #[clap(long)]
+        target_account: Option<AccountAddress>,
+    },
+    /// Online execution
+    Online {
+        /// Endpoint url to obtain the txn data,
+        /// e.g. `https://api.mainnet.aptoslabs.com/v1` for mainnet.
```

Comment:
what happened to the chars limit? Do we cap at 80/100? seems like first 2 lines can fit on one, if the third is that long

Reply:
done